rpc/core/types/responses.go

```diff
@@ -96,9 +96,9 @@ type ResultDumpConsensusState struct {
 }
 
 type ResultBroadcastTx struct {
-	Code abci.CodeType `json:"code"`
-	Data data.Bytes    `json:"data"`
-	Log  string        `json:"log"`
+	Code uint32     `json:"code"`
```

Comment:
address the `FIXME`s by using the abci response types here. Also probably good to use ResultBroadcastTxAsync, etc.